virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -5,13 +5,15 @@ include::modules/virt-document-attributes.adoc[]
 
 toc::[]
 
-{VirtProductName} provides layer-2 networking capabilities that allow you to connect
-virtual machines to multiple networks. You can import virtual machines with
-existing workloads that depend on access to multiple interfaces. You can also
-configure a PXE network so that you can boot machines over the network.
+You can attach virtual machines to multiple networks using Linux bridges.
+You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
 
-To get started, a network administrator configures a bridge network attachment definition
-for a namespace in the web console or CLI. Users can then create a NIC to attach pods and virtual machines in that namespace to the bridge network.
+To attach a virtual machine to an additional network:
+
+. Configure a bridge network attachment definition for a namespace in the web console or CLI.
+. Attach the virtual machine to the network attachment definition:
+.. In the web console, create a NIC for a new or existing virtual machine.
+.. In the CLI, include the network information in the virtual machine configuration.
```

Comment:
Do you have to do both of these steps, or is it one or the other? If it's the latter, I would use sub-bullets instead of a/b. If you do have to do both, these seem like they should be steps 2 & 3. (This comment assumes that you are keeping the numbered steps, but I'm not sure that you should.)

Reply:
Good call. Changed these to sub-bullets